packages/backend-app-api/src/wiring/BackendInitializer.ts

```diff
@@ -649,6 +670,24 @@ export class BackendInitializer {
 
     return pluginStartupBootFailureValue === 'continue';
   }
+
+  #getPluginModuleBootFailurePredicate(
```

Comment:
didnt know if this should be handled independently like this, or if it should overload the existing one with an optional moduleId.